config/api.yml

```diff
@@ -942,7 +942,7 @@
     :resource_actions:
       :get:
       - :name: read
-        :identifier: miq_request_show
+        :identifier: miq_request_view
```

Comment:
For some reason I thought that the identifier had a typo. I verified in miq_product_features.yml and it is indeed `miq_request_show`. Thanks for catching that.
That change has been undone.